cmd/kyma/alpha/enable/module/module.go

```diff
@@ -0,0 +1,146 @@
+package module
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/cli/internal/cli/alpha/module"
+	"k8s.io/apimachinery/pkg/types"
+	"time"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	cli.Command
+	opts *Options
+}
+
+func NewCmd(o *Options) *cobra.Command {
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module [name] [flags]",
+		Short: "Enables a module in the cluster or in the given Kyma resource",
+		Long: `Use this command to enable Kyma modules available in the cluster.
+
+### Detailed description
+
+For more information on Kyma modules, see the 'create module' command.
+
+This command enables an available module in the cluster. 
+A module is available when a ModuleTemplate is found for instantiating it with proper defaults.
+`,
+
+		Example: `Example:
+Enable "my-module" from "alpha"" channel in "default-kyma" from "kyma-system" namespace
+		kyma alpha enable module my-module -c alpha -n kyma-system -k default-kyma
+`,
+		RunE:    func(cmd *cobra.Command, args []string) error { return c.Run(cmd.Context(), args) },
+		Aliases: []string{"mod", "mods", "modules"},
+	}
+
+	cmd.Flags().DurationVarP(
+		&o.Timeout, "timeout", "t", 1*time.Minute, "Maximum time for the operation to enable a module.",
+	)
+	cmd.Flags().StringVarP(
+		&o.Channel, "channel", "c", "",
+		"The channel of the module to enable.",
+	)
+	cmd.Flags().StringVarP(
+		&o.Namespace, "namespace", "n", cli.KymaNamespaceDefault,
+		"The namespace of the Kyma to use. An empty namespace defaults to 'kyma-system'",
+	)
+	cmd.Flags().StringVarP(
```

Comment:
for the list command we use the positional arg 0 as the kyma name. I think we should either change the list command to also have the `k` flag or to align the positional argument here. WDYT?

Reply:
Good idea, but I'm leaning towards having positional arg as module name, since kyma name and namespace can be omitted using defaults.
Also if introducing `-k` for `list` I will default it as well to "default-kyma", you agree?